tests/models/test_gans.py

```diff
@@ -18,7 +18,9 @@ def test_gan(tmpdir, datadir, dm_cls):
 
     dm = dm_cls(data_dir=datadir, num_workers=0)
     model = GAN(*dm.size())
-    trainer = pl.Trainer(fast_dev_run=True, default_root_dir=tmpdir)
+    trainer = pl.Trainer(
+        fast_dev_run=True, logger=False, checkpoint_callback=False, max_epochs=1, default_root_dir=tmpdir
```

Comment:
I think we can standardize to use something like this.

Reply:
is it needed when you already have `fast_dev_run=True`?

Reply:
Well yes, This avoid some calls to logger, prevents checkpoints being stored.

Checkpoints can occupy unnecassary disk space and possible transfer from RAM to Memory causing `Memory Error`

I'm not sure if this will exactly solve the problem #526 But keeping this enabled will probably avoid these issues.

Reply:
Do we need `max_epochs=1` here? `fast_dev_run=True` ends right after the first batch in the first epoch, right?